tfx/extensions/google_cloud_big_query/utils.py

```diff
@@ -20,31 +20,8 @@
 from typing import Any, Dict, List
 
 import apache_beam as beam
-from apache_beam.io.gcp import bigquery
 from apache_beam.options import value_provider
 import tensorflow as tf
-from tfx.utils import telemetry_utils
-
-
-@beam.ptransform_fn
-@beam.typehints.with_input_types(beam.Pipeline)
-@beam.typehints.with_output_types(beam.typehints.Dict[str, Any])
-def ReadFromBigQuery(pipeline: beam.Pipeline,  # pylint: disable=invalid-name
-                     query: str) -> beam.pvalue.PCollection:
```

Comment:
Maybe changing it into `ReadFromBigQuery(pipeline:beam.Pipeline, query:str, **big_query_custom_config)` would be more appropriate.